client-dotnet/src/EvoMaster.Client.Util/JsonInspector.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace EvoMaster.Client.Util
+{
+    public class JsonInspector
+    {
+        public static bool ContainsKeyValue<T>(string json, string key, T value)
```

Comment:
provide some documentation for this function

Reply:
This class is removed in the latest push